skyjo.py

```diff
@@ -651,6 +734,42 @@ def greedy(skyjo: Skyjo) -> int:
     raise ValueError(f"Unexpected action {get_action(skyjo)}!")
 
 
+def start_round(skyjo: Skyjo, rng: Random = random) -> Skyjo:
+    """Take skyjo state an start round by flipping first card for each player."""
+    players = skyjo[3]
+    for _ in range(players):
+        skyjo = randomize(skyjo, rng=rng)
+        assert validate(skyjo)
+        skyjo = flip(skyjo, 0, 0, rotate=True, turn=False)
+        assert validate(skyjo)
+    return skyjo
+
+
+def apply_action(skyjo: Skyjo, action: SkyjoAction, rng: Random = random) -> Skyjo:
+    skyjo = randomize(skyjo, rng=rng)
```

Comment:
you unfortunately have to write this out every time except for when you are replacing an already-flipped card, in which case there's no random even necessary